dub.selections.json

```diff
@@ -12,7 +12,7 @@
 		"memutils": "1.0.9",
 		"mir-linux-kernel": "1.0.1",
 		"mustache-d": "0.1.5",
-		"openssl": "3.3.0",
+		"openssl": "3.3.4",
```

Comment:
I don't think this is going to fix the problem. Locally it still fails:
```
    Building vibe-d:tls 0.9.5: building configuration [openssl]
../../.dub/packages/openssl/3.2.2/openssl/source/deimos/openssl/opensslv.di(85,9): Error: static assert:  `-45 >= 0` is false
        static assert (build >= 0);
        ^
```
That's even after I update the versions below for vibe-core and vibe-d to latest - it apparently ignores that and gets 0.9.5.

Reply:
Also with `dub build` in my local `core` tour repo:
```
../../.dub/packages/vibe-http/1.3.1/vibe-http/source/vibe/http/internal/http2/exchange.d(329,42): Error: `@safe` function `vibe.http.internal.http2.exchange.handleHTTP2Request!(TCPConnection).handleHTTP2Request` cannot call `@system` function `vibe.internal.string.formatAlloc!long.formatAlloc`
        res.headers["Keep-Alive"] = formatAlloc(alloc, "timeout=%d", settings.keepAliveTimeout.total!"seconds"());
                                               ^
```

Reply:
I checked out vibe-http v1.3.1 separately and that builds fine without that error, so I don't know what's going on.